### PR TITLE
Connect 1963 update python

### PR DESCRIPTION
**Pull request by Majkluss**

### CODE REVIEW

File: spec/openapi.yaml [hunk 1]
```diff
@@ -508,7 +508,7 @@ paths:
           type: string
           enum: ["python"]
```
Comment (Lowentwickler) [inline spec/openapi.yaml:509]: Should not we rename it to `system` or `link` then?

Comment (Majkluss) [reply spec/openapi.yaml:509]: Good point, I'll find out

File: spec/openapi.yaml [hunk 2]
```diff
@@ -522,20 +522,30 @@ paths:
             application/json:
               schema:
                 oneOf:
-                  - $ref: "#/components/schemas/PythonPackageUpdateInfo"
+                  - $ref: "#/components/schemas/PrusaLinkPackage"
         204:
           description: No Content
+          headers:
+            Update-Available:
```
Comment (ondratu) [inline spec/openapi.yaml:529]: 204 mean, no update available right?!

**Review by ondratu — CHANGES_REQUESTED** [review]

File: spec/openapi.yaml [hunk 3]
```diff
@@ -1113,30 +1123,19 @@ components:
           type: string
           example: "Ba1kmCbifTa8X"
 
-    PythonPackageUpdateInfo:
+    PrusaLinkPackage:
       type: object
+      description: PrusaLink package version available to update
       properties:
         available_updates:
           type: array
```
Comment (Lowentwickler) [inline spec/openapi.yaml:1131]: We should either make it an object, not an array, or bring back name and version back into list.

File: spec/openapi.yaml (hunk 2 above)
Comment (Majkluss) [reply spec/openapi.yaml:529]: Yes, if there's no update available, this header has value "false".
I suppose you mean this header should not be here at all? I can change it.

**Review by ondratu — APPROVED** [review]

**Pull request merged**